setup.cfg

```diff
@@ -31,8 +31,9 @@ install_requires = Sphinx >= 2.1
 test =
     pytest >= 3.1.0
     typing_extensions >= 3.5
-    typed_ast >= 1.4.0; python_version < "3.8"
     dataclasses; python_version == "3.6"
+comment =
```

Comment:
this allows users to enable the comment support via ``sphinx-autodoc-typehints[comments]`` rather than have to know the typed ast library by name